lib/sensors/src/MoistureSensor.cpp

```diff
@@ -53,7 +52,7 @@ uint16_t MoistureSensor::Read_Moisture() {
 
   char buf[2];
 
-  uint16_t ret = 65535;
+  sensorReading = 65535;
 
   uint8_t counter = 10;  // initialize counter to break out of loop if reading isn't working (prevent infinite looping)
```

Comment:
number of attempts should be a const somewhere. also why aare we trying 10 times? we should try max of 2 imo. @ChristopherA057